### ISSUE TITLE
Error resolving plugins using Gradle 4.3

### ISSUE THREAD

**Issue by baron1405** [JetBrains/gradle-intellij-plugin#247]
Upgraded a plugin project from Gradle 4.2.1 to Gradle 4.3. The build failed attempting to resolve the external plugin CheckStyle-IDEA:5.12.0. This builds fine using Gradle 4.2.1. My plugin dependencies are:
```
android
CheckStyle-IDEA:5.12.0
copyright
CSS
gradle
Groovy
JavaScriptLanguage
junit
properties
uml
```
The error output is rather confusing but it ultimately boils down to the external plugin CheckStyle-IDEA:
```
* What went wrong:
Could not resolve all files for configuration ':compileClasspath'.
> Could not resolve org.jetbrains.plugins:android:2017.2.5.
  Required by:
      project :
   > Could not resolve org.jetbrains.plugins:android:2017.2.5.
      > inconsistent module metadata found. Descriptor: com.jetbrains:ideaIU:2017.2.5 Errors: bad group: expected='org.jetbrains.plugins' found='com.jetbrains'
        bad module name: expected='android' found='ideaIU'
> Could not resolve org.jetbrains.plugins:copyright:2017.2.5.
  Required by:
      project :
   > Could not resolve org.jetbrains.plugins:copyright:2017.2.5.
      > inconsistent module metadata found. Descriptor: com.jetbrains:ideaIU:2017.2.5 Errors: bad group: expected='org.jetbrains.plugins' found='com.jetbrains'
        bad module name: expected='copyright' found='ideaIU'
> Could not resolve org.jetbrains.plugins:CSS:2017.2.5.
  Required by:
      project :
   > Could not resolve org.jetbrains.plugins:CSS:2017.2.5.
      > inconsistent module metadata found. Descriptor: com.jetbrains:ideaIU:2017.2.5 Errors: bad group: expected='org.jetbrains.plugins' found='com.jetbrains'
        bad module name: expected='CSS' found='ideaIU'
> Could not resolve org.jetbrains.plugins:gradle:2017.2.5.
  Required by:
      project :
   > Could not resolve org.jetbrains.plugins:gradle:2017.2.5.
      > inconsistent module metadata found. Descriptor: com.jetbrains:ideaIU:2017.2.5 Errors: bad group: expected='org.jetbrains.plugins' found='com.jetbrains'
        bad module name: expected='gradle' found='ideaIU'
> Could not resolve org.jetbrains.plugins:Groovy:2017.2.5.
  Required by:
      project :
   > Could not resolve org.jetbrains.plugins:Groovy:2017.2.5.
      > inconsistent module metadata found. Descriptor: com.jetbrains:ideaIU:2017.2.5 Errors: bad group: expected='org.jetbrains.plugins' found='com.jetbrains'
        bad module name: expected='Groovy' found='ideaIU'
> Could not resolve org.jetbrains.plugins:JavaScriptLanguage:2017.2.5.
  Required by:
      project :
   > Could not resolve org.jetbrains.plugins:JavaScriptLanguage:2017.2.5.
      > inconsistent module metadata found. Descriptor: com.jetbrains:ideaIU:2017.2.5 Errors: bad group: expected='org.jetbrains.plugins' found='com.jetbrains'
        bad module name: expected='JavaScriptLanguage' found='ideaIU'
> Could not resolve org.jetbrains.plugins:junit:2017.2.5.
  Required by:
      project :
   > Could not resolve org.jetbrains.plugins:junit:2017.2.5.
      > inconsistent module metadata found. Descriptor: com.jetbrains:ideaIU:2017.2.5 Errors: bad group: expected='org.jetbrains.plugins' found='com.jetbrains'
        bad module name: expected='junit' found='ideaIU'
> Could not resolve org.jetbrains.plugins:properties:2017.2.5.
  Required by:
      project :
   > Could not resolve org.jetbrains.plugins:properties:2017.2.5.
      > inconsistent module metadata found. Descriptor: com.jetbrains:ideaIU:2017.2.5 Errors: bad group: expected='org.jetbrains.plugins' found='com.jetbrains'
        bad module name: expected='properties' found='ideaIU'
> Could not resolve org.jetbrains.plugins:uml:2017.2.5.
  Required by:
      project :
   > Could not resolve org.jetbrains.plugins:uml:2017.2.5.
      > inconsistent module metadata found. Descriptor: com.jetbrains:ideaIU:2017.2.5 Errors: bad group: expected='org.jetbrains.plugins' found='com.jetbrains'
        bad module name: expected='uml' found='ideaIU'
> Could not resolve org.jetbrains.plugins:CheckStyle-IDEA:5.12.0.
  Required by:
      project :
   > Could not resolve org.jetbrains.plugins:CheckStyle-IDEA:5.12.0.
      > inconsistent module metadata found. Descriptor: com.jetbrains:ideaIU:2017.2.5 Errors: bad group: expected='org.jetbrains.plugins' found='com.jetbrains'
        bad module name: expected='CheckStyle-IDEA' found='ideaIU'
        bad version: expected='5.12.0' found='2017.2.5'
```

**Comment by zolotov** [comment]
Are you on snapshot?

**Comment by zolotov** [comment]
Filed an issue: https://github.com/gradle/gradle/issues/3330

**Comment by baron1405** [comment]
Hi @zolotov I see that the Gradle issue has been closed and it looks like the suggested fix worked for you. Do you anticipate making a release with the fix for 4.3 soon?

**Comment by baron1405** [comment]
Ah, I see that Gradle 4.3.1 has been released and appears to fix the problem. I will try out 4.3.1 and report back.

**Comment by zolotov** [comment]
Should be fixed in 4.3.1 but I still want to make it work in 4.3, I think Gradle team did the right thing and I should to prepare to it in future

**Comment by baron1405** [comment]
Sounds good. I have verified that 4.3.1 does fix the issue.
